seed/csharp-model/multi-line-docs/src/SeedMultiLineDocs/User/User.cs

```diff
@@ -11,7 +11,7 @@ public record User
     public required string Id { get; set; }
 
     /// <summary>
-    /// The user's name. This name is unique to each user. A few examples are included below:
+    /// The user&#39;s name. This name is unique to each user. A few examples are included below:
```

Comment:
it's looking like lodash's escape might be a little overkill for us -- from my understanding, lodash is going to escape the following chars: `& < > " '`

But I think that we only need carats to be escaped

Can we roll our own method only escapes carats?

Reply:
Yes, I was on the fence about that. Let's do it ourselves.

Reply:
@dcb6 The code has been updated with our own escaping.